r/R/dataset.R

```diff
@@ -15,6 +15,31 @@
 # specific language governing permissions and limitations
 # under the License.
 
+#' @usage NULL
```

Comment:
This should not go first in the file. Move it to after Dataset and before Source.

Still needs docs added. Could you please take a stab at that? I could, but I'm not sure I'd get the reasoning right.

Reply:
I'll fix this in order to remove my objections to merging so that Krisz can build on this separately, and I'll follow up later with my proposal to remove `Dataset$create`